utils/index.sh

```diff
@@ -0,0 +1,132 @@
+#!/bin/bash
+
+set -exo pipefail
+
+setup(){
+    # Generate a uuid
+    export UUID=${UUID:-$(uuidgen)}
```

Comment:
I think we shouldn't generate UUID if it's not defined. It could be confusing to see this document with a UUID different from the benchmark's indexed documents.

Maybe we should hang when is not defined instead?


```
$ ./index.sh 
error: UUID env var not defined
```

Reply:
not sure the significance of the UUID here. We index after install, and other tasks -- which don't have a "uuid" hence the uuidgen. I don't have a strong opinion here.

Reply:
I think we should keep this statement as is for the below reasons.
* In PROW `UUID` will either get generated as a part of [run.sh](https://github.com/cloud-bulldozer/e2e-benchmarking/blob/master/workloads/kube-burner/common.sh#L18) or even our tools `k8s-netperf` and `kube-burner` have checks to generate one if already not present. So for PROW jobs it will never fall in the `uuidgen` case in this script.
* And when it comes to airflow, I think that is where the `uuidgen` in this script comes into play. Right after the [install](https://github.com/cloud-bulldozer/airflow-kubernetes/blob/master/dags/openshift_nightlies/scripts/install/cloud.sh) step there won't be any `UUID` present in our system, and when the execution reaches either of `index-install` or `benchmark` step whichever executes first we have an `UUID` generated and maintained same across the system.

So I believe this statement is of no harm and is useful in some of the edge cases for our airflow runs like it is present as of [now](https://github.com/cloud-bulldozer/airflow-kubernetes/blob/master/dags/openshift_nightlies/scripts/index.sh#L16) .